### PR TITLE
55 Usability Changes

### PR DESCRIPTION
**Pull request by Cinnazeyy**

### CODE REVIEW

File: src/main/java/com/alpsbte/plotsystem/commands/plot/CMD_Plot_Feedback.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public void onCommand(CommandSender sender, String[] args) {
                     return;
                 }
 
-                if(plot.getPlotOwner().getUUID().equals(getPlayer(sender).getUniqueId()) || plot.getPlotMembers().stream().anyMatch(m -> m.getUUID().equals(getPlayer(sender).getUniqueId())) || getPlayer(sender).hasPermission("alpsbte.plot.review")) {
+                if(plot.getPlotOwner().getUUID().equals(getPlayer(sender).getUniqueId()) || plot.getPlotMembers().stream().anyMatch(m -> m.getUUID().equals(getPlayer(sender).getUniqueId())) || getPlayer(sender).hasPermission("plotsystem.plot.review")) {
```
Comment (LordTuxn) [inline src/main/java/com/alpsbte/plotsystem/commands/plot/CMD_Plot_Feedback.java:66]: Please move to a separate commit

**Review by LordTuxn — CHANGES_REQUESTED** [review]

File: src/main/java/com/alpsbte/plotsystem/commands/plot/CMD_Plot_Feedback.java (hunk 1 above)
Comment (Cinnazeyy) [reply src/main/java/com/alpsbte/plotsystem/commands/plot/CMD_Plot_Feedback.java:66]: waaay too much effort for 1 line
